### ISSUE TITLE
Indicate chapter inactivity

### ISSUE THREAD

**Issue by jcn** [awesomefoundation/awesomebits#94]
We have a number of chapters that are not currently active, which is not great if they're receiving grant applications but aren't actually responding to people. An inactive chapter should:

- [x] Not be listed in the application dropdown
- [x] Not have an application button on the page
- [x] Have text on the main chapter page with a link to the join@awesomefoundation.org email
- [x] Not be counted in the "X CHAPTERS" text on the main page
- [x] Be excluded from the main chapter list at the top of the page

Note: we are going to explicitly keep the chapters listed on the current `/chapters` page so they will continue to get indexed by search engines. At some point in the future we can redesign that page and we can rethink the display of those chapters at that point.

**Comment by jcn** [comment]
> Have text on the main chapter page with a link to the "getting started" spreadsheet

@avicaplan We're not using the "getting started" spreadsheet any more, right? For an inactive chapter, is the best thing to do to have people contact join@awesomefoundation.org and you can work with them that way?

**Comment by avicaplan** [comment]
Correct!

**Comment by jcn** [comment]
@avicaplan I'm thinking about language based on the language you already use on the site:

"**This chapter is currently inactive.** An inactive chapter is one which has not given a grant in six months. If you are interested in re-starting this chapter, please contact us at join@awesomefoundation.org."

How does that sound?

**Comment by avicaplan** [comment]
I wonder if we might want to more or less recycle the language from the About Us page:

If you're interested in [starting your own] *restarting this* chapter, which involves assembling a group of at least ten "trustees" willing to each contribute $100 (or the local equivalent) every month toward the grant, please get in touch at join@awesomefoundation.org.

Or do you think that's too wordy?

**Comment by jcn** [comment]
Seems too wordy to me, especially given that the main point of that page (in my mind) is as a reference point at least for now. Maybe we can keep the shorter text and see whether we get any emails at all.

**Comment by jcn** [comment]
@Chrysaora @avicaplan I've pushed a version of this to http://staging.awesomefoundation.org for review. You should be able to log in (or reset your password) to be able to log in as an admin. There's a new checkbox on the Chapter edit page for making a chapter inactive and then you can make sure that everything works properly (see the spec as defined in the description of this issue above).

This does NOT automatically disable a chapter if it hasn't funded a project in 6 months - I want to work on that as a separate issue.

**Comment by avicaplan** [comment]
Regarding the language, sounds good!

In terms of the staging site, I can't seem to log in -- my password from the live site doesn't work, and when I try a password reset on the staging site it directs me to reset on the live site (hope that made sense).

**Comment by jcn** [comment]
> 
All you have to do is change the url in the reset link to "staging" instead of "www" and it should work.

**Comment by jcn** [comment]
@avicaplan If you can't get that working, I made the "Food" chapter on staging inactive:

http://staging.awesomefoundation.org/en/chapters/food

**Comment by avicaplan** [comment]
Got in, but I don't seem to have admin (or even dean) rights on the staging site. The Food chapter looks good. Seeing the text there, I do wonder though if rather than "contact us" we might say "contact join@awesomefoundation.org." Otherwise, who is the "us"? Is it the inactive chapter? But no big deal either way. Also, I think on reading this now that I made a tiny grammatical mistake when I first wrote the text, and it should be "that" not "which." :)

**Comment by jcn** [comment]
Ok based on the fact that I already have some of the translations in I'm going to go with the existing text (but I'll fix the grammar) and I'll push this live and we can start using it!

**Comment by avicaplan** [comment]
Sounds good! Thanks for making this happen.

**Comment by avicaplan** [comment]
(And happy to go in and switch things over from the workaround to the proper system, if you let me know when this is live).

**Comment by jcn** [comment]
@avicaplan OK, this is live so you can go in and switch inactive chapters over to this system. If you ever need to get at an inactive chapter, http://www.awesomefoundation.org/en/chapters will list all chapters (not just active ones).

**Comment by avicaplan** [comment]
Done! By the way, number of chapters on the main page updated but number of countries didn't. We may also want to change now where the column breaks are in the chapter list -- could also be an opportunity to reorder alphabetically and put worldwide at the end.

**Comment by jcn** [comment]
@avicaplan OK, I've fixed the country bug and the other issue can be discussed in #165. It still considers "Worldwide" chapters as their own country, but I'll just count this as a feature, not a bug. Considering this issue really closed now.
